package.json

```diff
@@ -35,7 +35,8 @@
 		"regenerator-runtime": "^0.13.5",
 		"resize-observer-polyfill": "^1.5.1",
 		"setimmediate": "^1.0.5",
-		"use-resize-observer": "^6.1.0"
+		"use-resize-observer": "^6.1.0",
+		"uuid": "^3.4.0"
```

Comment:
there is nanoid dependency already, we can move it to  dependencies from devDependencies